src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentEditAction.java

```diff
@@ -58,6 +63,9 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                 courseId, feedbackSessionName, null, instructor.email, null, new Date(),
                 new Text(commentText), response.giverSection, response.recipientSection);
         feedbackResponseComment.setId(Long.parseLong(feedbackResponseCommentId));
+        List<StudentAttributes> students = logic.getStudentsForCourse(courseId);
+        List<InstructorAttributes> instructors = logic.getInstructorsForCourse(courseId);
+        CourseRoster roster = new CourseRoster(students, instructors);
```

Comment:
I really don't like how editing a single comment will now require retrieving the entire course roster (including all students and all instructors of a course), just to get the name of the instructor who just edited the comment (which can already be found inside the `instructor` variable). In addition to it being horribly inefficient, this spells trouble for large courses.

Please change `students` to `null` and `instructors` to a list containing nothing but the current `instructor`, as that is really all that is needed.